test/util.py

```diff
@@ -0,0 +1,55 @@
+import os
+import pandas as pd
+
+outdir = 'baseline/results'
+if not os.path.exists(outdir):
+  os.makedirs(outdir)
+
+df_testing = pd.read_csv('project_testing/testing_baseline.csv')
+df_testing['building_id'] = df_testing['building_id'].apply(lambda x: 'project_testing-{}.osw'.format('%04d' % x))
+df_testing.insert(1, 'color_index', 0)
+
+df_national = pd.read_csv('project_national/national_baseline.csv')
+df_national['building_id'] = df_national['building_id'].apply(lambda x: 'project_national-{}.osw'.format('%04d' % x))
+df_national.insert(1, 'color_index', 1)
+
+frames = [df_national, df_testing]
+df = pd.concat(frames)
+df = df.rename(columns={'building_id': 'OSW'})
+del df['job_id']
+
+build_existing_models = []
+simulation_output_reports = ['color_index']
+qoi_reports = []
+
+for col in df.columns.values:
+  if 'applicable' in col:
+    continue
+
+  elif col.startswith('build_existing_model'):
+    build_existing_models.append(col)
+  elif col.startswith('simulation_output_report'):
+    simulation_output_reports.append(col)
+  elif col.startswith('qoi_report'):
+    qoi_reports.append(col)
+
+# results_characteristics.csv
+results_characteristics = df[['OSW'] + build_existing_models]
+for col in results_characteristics.columns.values:
+  results_characteristics = results_characteristics.rename(columns={col: col.replace('build_existing_model.', '')})
+
+results_characteristics = results_characteristics.set_index('OSW')
+results_characteristics = results_characteristics.reindex(sorted(results_characteristics), axis=1)
+results_characteristics.to_csv('baseline/results/results_characteristics.csv')
+
+# results_output.csv
+results_output = df[['OSW'] + simulation_output_reports + qoi_reports]
+results_output = results_output.dropna(how='all', axis=1)
+results_output = results_output.round(1)
+for col in results_output.columns.values:
+  results_output = results_output.rename(columns={col: col.replace('simulation_output_report.', '')})
```

Comment:
1. This will need to change to "report_simulation_output" soon.
2. Better yet, can't you write the code so that you just generically strip out strings before the ".", so you don't have to hardcode measure names here?
3. Even better, I would prefer these strings not be stripped out at all, so that outputs are grouped together logically. With this change, the plots jump around from energy to cost multipliers to qoi back to energy...

Reply:
This will all go away when `restructure-v3` is merged: prefixes will be maintained.

Reply:
@shorowit Maybe you should review https://github.com/NREL/resstock/pull/706 as well?